src/features/Classes/ClassDetailPage/InstructorCard.jsx

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+import { useSelector } from 'react-redux';
+
+import { formatDateRange } from 'react-paragon-topaz';
+import { Spinner, ProgressBar } from '@edx/paragon';
+
+import { RequestStatus } from 'features/constants';
+
+const InstructorCard = () => {
+  const classes = useSelector((state) => state.common.allClasses);
+  const [classInfo] = classes.data;
```

Comment:
What if

```suggestion
   const [classInfo] = useSelector((state) => state.common.allClasses?.data);
```

Reply:
I have to access the status, is longer the line if a have to access the useselector for that